docs/appendix/glossary/resource.md

```diff
@@ -2,10 +2,12 @@
 title: Resource
 id: resource
 full_link:
-short_description: Resources are individual, addressable elements of a robot (RDK definition) operated by parts.
+short_description: Resources are individual, addressable elements of a robot such as components or services.
 aka:
 ---
 
-Resources are individual, addressable elements of a robot (RDK definition) operated by parts.
+Resources are individual, addressable elements of a robot.
 
-Parts operate two types of Resources: physical {{< glossary_tooltip term_id="component" text="components" >}} and software services. Each part has local resources, and also surfaces remote resources when a {{< glossary_tooltip term_id="remote" text="remote">}} is established to another part. The capabilities of each resource are exposed through the part’s API.
+{{< glossary_tooltip term_id="part" text="Parts" >}} operate two types of Resources: physical {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="software services" >}}.
```

Comment:
```suggestion
{{< glossary_tooltip term_id="part" text="Parts" >}} operate two types of resources: physical {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="software services" >}}.
```
Need to decide whether modules, remotes and/or processes count as resources (see comment above), but maybe leave for now idk

Reply:
yeah let's leave for now if we're not sure